prometheus/config/node_exporter/textfile_collectors/ipmitool/init.sls

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls
+
+{#- Get the `tplroot` from `tpldir` #}
+{%- set tplroot = tpldir.split('/')[0] %}
+{%- from tplroot ~ "/map.jinja" import prometheus with context %}
+
+{%- set config = prometheus.node_exporter.textfile_collectors.ipmitool %}
+{%- set dir = prometheus.service.node_exporter.args.get('collector.textfile.directory') %}
+{%- set script = prometheus.dir.textfile_exporters ~ '/ipmitool' %}
+
+prometheus-exporters-node-textfile_collectors-ipmitool-pkg:
+  pkg.installed:
+    - name: {{ config.pkg }}
+
+prometheus-exporters-node-textfile_collectors-ipmitool-script:
+  file.managed:
+    - name: {{ script }}
+    - source: salt://prometheus/config/node_exporter/textfile_collectors/files/ipmitool
+    - mode: 755
+    - require:
+      - file: prometheus-node_exporter-textfile_collectors-dir
+
+{%-   set cmd_prefix = 'awk -f ' if grains.os_family in ['FreeBSD'] else '' %}
+
+prometheus-exporters-node-textfile_collectors-ipmitool-cronjob:
+  cron.present:
+    - identifier: prometheus-exporters-node-textfile_collectors-ipmitool-cronjob
+    - name: cd {{ dir }} && LANG=C ipmitool sensor | {{ cmd_prefix }}{{ script }} > .ipmitool.prom$$; mv .ipmitool.prom$$ ipmitool.prom
```

Comment:
Should LANG be configurable under `prometheus.config.node-exporter.lang`?

Should `ipmitool` command be configurable via pillars?  See http://ipmiutil.sourceforge.net/docs/ipmisw-compare.htm . 

We could raise as (enhancement) issue - for most users this is enough.

Reply:
`LANG=C` forces output which can be easily parsed.

Regarding `ipmitool`: If you change the command, the output changes. Which means: change the command, change the script. So that's stuff for another PR. :-)

Reply:
sure - a nest of snakes, in other words.